### PR TITLE
Origin/feature/20/home page content

### PR DESCRIPTION
**Pull request by KaliszukTomasz**
Home page with working header-bar is ready to add to project.

### CODE REVIEW

File: src/app/header-bar/header-bar.component.ts [hunk 1]
```diff
@@ -13,28 +13,52 @@ export class HeaderBarComponent implements OnInit {
...
+
+  loggedAddress: string[] = [];
   isLoggedInToMetamask: boolean = false;
   isLoggedAsAdmin: boolean = false;
 
-  constructor(private web3service: Web3Service) { 
-    web3service.bootstrapWeb3();
-    this.accountAddress=web3service.getAccountAddress();
+  constructor(private web3Service: Web3Service) {
+    web3Service.bootstrapWeb3();
```
Comment (lukaszkinder) [inline src/app/header-bar/header-bar.component.ts]: all this move to noOnInit

File: src/app/util/web3.service.ts [hunk 2]
```diff
@@ -35,16 +43,18 @@ export class Web3Service {
       // Hack to provide backwards compatibility for Truffle, which uses web3js 0.20.x
       Web3.providers.HttpProvider.prototype.sendAsync = Web3.providers.HttpProvider.prototype.send;
       // fallback - use your lfallback strategy (local node / hosted node + in-dapp id mgmt / fail)
-      this.web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));
```
Comment (lukaszkinder) [inline src/app/util/web3.service.ts]: would be good to keep url and port no in environment properties

File: src/app/util/web3.service.ts [hunk 3]
```diff
@@ -76,14 +86,49 @@ export class Web3Service {
...
+
+  public getCurrentBalance(): Observable<string> {
+    return of(this.currentBalance);
+  }
+
+  public getBalanceOfAddressSubject(): Observable<string>{
+    return this.balanceOfAddressSubject.asObservable();
+  }
+
+  public getCurrentBalanceOfAddress(): void {
```
Comment (lukaszkinder) [inline src/app/util/web3.service.ts]: is getCurrentBalance method needed if this one is implemented?

File: src/app/welcome-menu/welcome-menu.component.ts [hunk 4]
```diff
@@ -1,4 +1,7 @@
 import { Component, OnInit } from '@angular/core';
+import { Web3Service, ADMIN_ADDRESS } from '../util/web3.service';
+
+const adminAddress: string = '0x9ab903e4f6da82e20e265b12f11c08aa62f82cc3';
```
Comment (lukaszkinder) [inline src/app/welcome-menu/welcome-menu.component.ts]: this should be placed in only one place

File: src/app/header-bar/header-bar.component.ts (hunk 1 above)
Comment (KaliszukTomasz) [reply src/app/header-bar/header-bar.component.ts]: Done

File: src/app/util/web3.service.ts (hunk 2 above)
Comment (KaliszukTomasz) [reply src/app/util/web3.service.ts]: Ok

File: src/app/util/web3.service.ts (hunk 3 above)
Comment (KaliszukTomasz) [reply src/app/util/web3.service.ts]: Of course not.

**Pull request merged**